kolibri/plugins/management/assets/src/actions.js

```diff
@@ -25,19 +25,30 @@ const samePageCheckGenerator = require('kolibri.coreVue.vuex.actions').samePageC
  * the API to state in the Vuex store
  */
 
-function _userState(data) {
-  // assume just one role for now
+function _stateUser(apiUserData) {
+  // handle role representation
   let kind = UserKinds.LEARNER;
-  if (data.roles.length && data.roles[0].kind === 'admin') {
-    kind = UserKinds.ADMIN;
-  }
+
+  // makes kind the highest ranking role
+  apiUserData.roles.forEach(role => {
+    // using a switch statement. Checks all in order of heirarchy
+    switch(role.kind){
+      // sets role to admin 
+      case UserKinds.ADMIN || UserKinds.SUPERUSER:
+        kind = UserKinds.ADMIN;
+        break;
+      case UserKinds.COACH:
+        if(kind != UserKinds.ADMIN) kind = UserKinds.COACH;
+        break;
+    }
+  });
```

Comment:
I mean we can retain the `SUPERUSER` info, but I haven't seen it in any of the mock ups and judging by how everyone was concerned that a `SUPERUSER` could delete themselves, I assumed that there wasn't much differentiating a superuser from an admin. 

I'll take a look at making this more easily reasonable.